tests/arch/x86/nmi/src/main.c

```diff
@@ -17,13 +17,13 @@
 
 static volatile int int_handler_executed;
 
-extern uint8_t z_x86_nmi_stack[];
+extern uint8_t z_x86_nmi_stack0[];
```

Comment:
The changes here are not in-line with the commit message. Could you re-phrase the commit message?

Reply:
Please also spell check word  `varible` in commit message and subject

Reply:
Still don't get why you've included a whole patch in the PR just to glue a "0" onto the end of a variable that seems perfectly clear on its own?   Not a big deal, just weird.